source/llassetgen/include/llassetgen/Image.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <llassetgen/Vec2.h>
+
+#include <limits>
+#include <string>
+#include <memory>
+
+struct FT_Bitmap_;
+struct png_struct_def;
+
+namespace llassetgen {
```

Comment:
We don't indent within a namespace and place the opening brace on the next line.

Reply:
In this project we use a formatting tool which places the braces like that.